src/Http/Resources/V1/Admin/Sale/ShipmentResource.php

```diff
@@ -18,7 +19,7 @@ public function toArray($request)
     {
         return [
             'id'               => $this->id,
-            'status'           => $this->status,
+            'order_status'     => $this->when($this->order_id, new OrderResource($this->order))->status,
```

Comment:
Make this `order` key and pass the full order resource.

Reply:
fixed

Reply:
Here, When I added full order resource then it takes to much time and at the end its giving **error**. 

![Screenshot from 2023-05-24 12-10-40](https://github.com/bagisto/rest-api/assets/121479029/f7146cd0-8ba0-47ed-9928-e0579c00644e)

Reply:
Fixed